README.md

```diff
@@ -1,43 +1,41 @@
 [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
 ## Instructions
 
-1. Create an Auth0 account.
+1. Create an Auth0 account, if you haven't already at [auth0.com](https://auth0.com/).
 
-2. Create an Auth0 application (`Applications` > `Applications`).
+2. Create an Auth0 application by going to the `Applications` page under the `Applications` tab on the left.
 
-   - Choose `Regular Web Application`
-   - Go to the `Settings` tab
-   - Copy all the following values to the indicated fields:
+   - Choose to create a `Regular Web Application`.
+   - Take note of your `domain`, you'll need it later. It typically looks like this: `dev-by9w1mxg.us.auth0.com`.
+   - Go to the `Settings` tab and scroll down to the fill in the following fields:
      - Allowed Callback URLs: http://localhost:3000/api/auth/callback
      - Allowed Logout URLs: http://localhost:3000/
      - Allowed Web Origins: http://localhost:3000/
      - Allowed Origins (CORS): http://localhost:3000/
-   - Take note of your `domain`, you'll need it later. It typically looks like this: `dev-by9w1mxg.us.auth0.com`.
-   - Save the application.
+   - Scroll down to the very bottom of page and click Save Changes; the application doesn't automatically save itself!
 
-3. Create a TakeShape project using the pattern in this repo.
+3. Create a TakeShape project using the pattern in this repo. This button will deploy the project for you:
 
    - <a target="_blank" href="https://app.takeshape.io/add-to-takeshape?repo=https://github.com/takeshape/takeshape-starter-auth0/tree/main/.takeshape/pattern"><img alt="Deploy To TakeShape" src="https://camo.githubusercontent.com/1b580e3ce353d235bde0f376ca35b0fb26d685f3750a3013ae4b225dd3aaf344/68747470733a2f2f696d616765732e74616b6573686170652e696f2f32636363633832352d373062652d343331632d396261302d3130616233386563643361372f6465762f38653266376264612d306530382d346564652d613534362d3664663539626536613862622f4465706c6f79253230746f25323054616b65536861706525343032782e706e673f6175746f3d666f726d6174253243636f6d7072657373" width="205" height="38" data-canonical-src="https://images.takeshape.io/2cccc825-70be-431c-9ba0-10ab38ecd3a7/dev/8e2f7bda-0e08-4ede-a546-6df59be6a8bb/Deploy%20to%20TakeShape%402x.png?auto=format%2Ccompress" style="max-width:100%;"></a>
 
 4. Create an Auth0 service in your new TakeShape project.
 
-   - Go to `Schema`, then click `Connect Service`.
-   - Use your Auth0 `domain` from the earlier step.
+   - Go to the `Schema` tab, then click `Connect Service`.
+   - Type in your Auth0 `domain` from the earlier step.
    - Take note of the `audience` from the TakeShape config screen, you'll need it later.
    - Save the service.
 
 5. Set up your TakeShape roles.
 
    - Go to `Settings` > `Roles`
    - Create a new custom role named `auth0`. Roles are case-sensitive, so be sure it is all lowercase.
-   - Give your role the following permissions:
+   - Check the following permissions:
```

Comment:
"Check" could interpreted as "examine" instead of "mark the checkbox," so I added some extra clarity here.

```suggestion
   - Use the checkboxes to enable the following permissions:
```